src/components/dataItem/DataItemSelect.js

```diff
@@ -14,6 +14,7 @@ export class DataItemSelect extends Component {
         className: PropTypes.string,
         label: PropTypes.string,
         value: PropTypes.string,
+        allowNone: PropTypes.bool,
```

Comment:
When would we not allow None?

Reply:
For thematic layers: 

![skjermbilde 2018-11-21 kl 13 20 09](https://user-images.githubusercontent.com/548708/48840911-3aa89380-ed90-11e8-98ea-c403f6df7bac.png)